Tootbot/Source/View Models/StatusCellViewModel.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright (C) 2017 Tootbot Contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import CoreData
+import Alamofire
+import ReactiveSwift
+import ReactiveCocoa
+import Result
+
+class StatusCellViewModel {
+    let displayName = MutableProperty<String?>(nil)
+    let userName = MutableProperty<String?>(nil)
+    let attachments = MutableProperty<[UIImage?]>([])
+    let boosted = MutableProperty<Bool>(false)
+    let boostedByName = MutableProperty<String?>(nil)
+    let createdAtDate = MutableProperty<Date?>(nil)
+    let hasAttachments = MutableProperty<Bool>(false)
+
+    let contentSignalProducer: SignalProducer<NSAttributedString, NoError>
+    let avatarSignalProducer: SignalProducer<UIImage, ImageCacheController.Error>
+
+    private let status: Status
+    private let imageCacheController = ImageCacheController()
```

Comment:
It might make sense to initialize the view model _with_ an `ImageCacheController` instead of creating a new one per view model? We could put the cache into the "parent" view model and inject it into here.